src/lgtm/entrypoint.sh

```
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo "TODO: start Loki, Tempo, Prometheus, Pyroscope, OpenTelemetry Collector and Grafana
```

Shellcheck (0.10.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)